src/github/mod.rs

```diff
@@ -210,6 +193,52 @@ pub fn handle_user(conn: &PgConnection, user: &GitHubUser) -> DashResult<()> {
     Ok(())
 }
 
+pub fn nuke_reactions(repo: &str) -> DashResult<()> {
+    let issue_reactions = SETUP.prohibited_issue_reactions(repo);
+    let comment_reactions = SETUP.prohibited_comment_reactions(repo);
+
+    if issue_reactions.is_empty() && comment_reactions.is_empty() {
+        // All reactions are permitted, bail early to avoid work.
+        return Ok(());
+    }
+
+    // Fetch all issues with some forbidden reaction:
+    let issues = GH.open_issues_with_reactions(repo)?.filter(|rifj|
+        issue_reactions.iter().any(|&react| rifj.reactions.has_reaction(react))
+    );
```

Comment:
Just noticed a bug here. If the issue does not have an unwanted reaction, then the reactions for the comments won't be processed at all.